File: cloud/gateway-server/src/main/java/pers/acp/springcloud/gateway/GateWayServerApplication.java
package pers.acp.springcloud.gateway;

import org.springframework.boot.SpringApplication;
import org.springframework.cloud.client.SpringCloudApplication;

/**
 * @author zhangbin by 2018-3-10 20:45
 * @since JDK 11
 */
@SpringCloudApplication
public class GateWayServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(GateWayServerApplication.class, args);
    }

}

File: common/gateway-server/src/main/kotlin/pers/acp/admin/gateway/GateWayServerApplication.kt
package pers.acp.admin.gateway

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

/**
 * @author zhangbin by 2018-3-10 20:45
 * @since JDK 11
 */
@SpringBootApplication
@EnableDiscoveryClient
class GateWayServerApplication

fun main(args: Array<String>) {
    runApplication<GateWayServerApplication>(*args)
}
